test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require 'test_helper'
+
+class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
+  driven_by :selenium, using: :chrome, screen_size: [1400, 1400]
+end
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/177002506-48a9ca4c-9f9d-4c87-82c1-be652fad72cc.png)

- [ ] Kindly note that you need to write unit and integration tests in your app as the requirements say.

![image](https://user-images.githubusercontent.com/65068771/177002673-99ac32c3-a363-4302-b86a-2bd00bb6967e.png)

Reply:
@andersonlebon, Thanks for your review. we have implemented it.